flocker/cli/test/test_cli_script.py

```diff
@@ -12,7 +13,7 @@ class FlockerCLITests(FlockerScriptTestsMixin, TestCase):
     command_name = u'flocker'
 
 
-class CLIOptionsTests(StandardOptionsTestsMixin, SynchronousTestCase):
```

Comment:
Yeah I'm sorta 50/50 on if it's worth doing. Definitely an improvement to the code, but arguably not the most effective use of engineer time? On the other hand if it appears in the review column I'll gladly review it.